language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -89,13 +90,19 @@ optionsParser = Options
     <*> strOption
         (  short 'o'
         <> metavar "DIR"
-        <> help "Output directory for the generated TypeScript files"
+        <> help "Output directory for the generated packages"
         )
     <*> optional (strOption
         (  short 'p'
         <> metavar "PACKAGE-JSON"
         <> help "Path to an existing 'package.json' to update"
         ))
+    <*> (Scope <$> strOption
+        (  short 's'
+        <> metavar "SCOPE"
+        <> value "@daml.js"
```

Comment:
What’s the point of including the `@` here? Can a scope ever not contain an `@`? My understanding was that the `@` sign is not part of the scope name but just syntax to differentiate the scope from the name. So it seems more sensible to not include it here.

Reply:
@cocreature in https://github.com/digital-asset/daml/pull/4818#discussion_r388145444, @hurryabit indicated that he'd prefer to see the '@'. i'm easy on this point.

Reply:
That was about the internals not the UX for the CLI argument afaict. Either way, if @hurryabit prefers this I don’t feel too strongly about this.

Reply:
As @cocreature said, I preferred it for the internals. If we required our users to pass the `@` to the CLI argument, we also need to check that they actually do it, which we currently don't. That said, I prefer if our would not need to pass in the `@` symbol but just `daml.js`.

Reply:
https://github.com/digital-asset/daml/pull/4939